cosmos/log.py

```diff
@@ -14,8 +14,10 @@
     "%(log_color)s%(message)s%(reset)s"
 )
 
+LOGGER_NAME_TEMPLATE = "astronomer-cosmos-{}"
 
-def get_logger(name: str | None = None) -> logging.Logger:
+
+def get_logger(name: str) -> logging.Logger:
```

Comment:
We need a unit test to cover this change

Reply:
Sure, will add a simple one tomorrow

Reply:
@tatiana Done